src/linear_algebra/bilinear_form.lean

```diff
@@ -38,26 +43,20 @@ open_locale big_operators
 universes u v w
 
 /-- A bilinear form over a module  -/
```

Comment:
```suggestion
/-- A bilinear form over a semimodule. -/
```
The module docstring should also be updated as well. Mentioning semimodules everywhere instead of modules could make things more obscure for mathematicians, so maybe just one line mentioning that certain things are defined to work over semimodules is fine.

Reply:
I mentioned in the module docs that we'll use "module" to mean "(semi)module".